### PR TITLE
Add simple distcp job publishing to S3 as an example

### PR DESCRIPTION
**Pull request by autumnust**
Tested From local.

### CODE REVIEW

File: gobblin-example/src/main/resources/distcpToExternal.job [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+job.name=GobblinDatabaseCopyTest
+job.description=Gobblin job for copy to S3
+
+
+
+# target publishing location for copy
+data.publisher.final.dir=/
+
+gobblin.dataset.profile.class=gobblin.data.management.copy.CopyableGlobDatasetFinder
+gobblin.dataset.pattern=/Users/lesun/Documents/localGobblinTest/localtest/*
```
Comment (ibuenros) [inline gobblin-example/src/main/resources/distcpToExternal.job]: Please make this not specific to your system.

File: gobblin-example/src/main/resources/distcpFromExternal.job [hunk 2]
```diff
@@ -0,0 +1,60 @@
+# ====================================================================
```
Comment (ibuenros) [inline gobblin-example/src/main/resources/distcpFromExternal.job]: Rename file to `distcpFromS3`?

File: gobblin-example/src/main/resources/distcpFromExternal.job [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+# target publishing location for copy 
+# The folder containing result files that will show up in the s3
+data.publisher.final.dir=<Full local(target) FS Path>
+
+gobblin.dataset.profile.class=gobblin.data.management.copy.CopyableGlobDatasetFinder
+
+#e.g. s3a://gobblinoutput/
+source.filebased.fs.uri=<Full Remote FS Path>
+gobblin.dataset.pattern=<Remote Dataset Pattern>
+#gobblin.dataset.pattern=<Dataset location pattern based on your filesystem>
```
Comment (ibuenros) [inline gobblin-example/src/main/resources/distcpFromExternal.job]: What is this used for?

File: gobblin-example/src/main/resources/distcpFromExternal.job [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+source.filebased.encrypted.fs.s3a.secret.key=<Encrypted Secret key>
+fs.s3a.buffer.dir=<Local Copy Bufer>
+# e.g. file:///
+writer.fs.uri=<Local FS Uri>
+gobblin.copy.recursive.update=true
+
+#You can disable ssl if necessary
+#fs.s3a.connection.ssl.enabled=false
+
+#Some encryption-related stuff, the content is 'gobblin'
```
Comment (ibuenros) [inline gobblin-example/src/main/resources/distcpFromExternal.job]: is `the content is 'gobblin'` necessary?

Comment (autumnust) [inline gobblin-example/src/main/resources/distcpFromExternal.job]: No, just an example.

File: gobblin-example/src/main/resources/distcpFromExternal.job (hunk 3 above)
Comment (autumnust) [inline gobblin-example/src/main/resources/distcpFromExternal.job]: ditto

**Comment by abti** [comment]
Issue: https://issues.apache.org/jira/browse/GOBBLIN-8

Please update your PR title with following prefix: [GOBBLIN-8]

**Pull request closed without merging**